moduleframework/helpers/container_helper.py

```diff
@@ -37,41 +37,46 @@ def __init__(self):
         set basic object variables
         """
         super(ContainerHelper, self).__init__()
-        static_name="testcontainer"
+        static_name = "testcontainer"
         self.tarbased = None
-        self.jmeno = None
+        self.name = None
         self.docker_id = None
-        self.icontainer = self.get_url()
-        if not self.icontainer:
+        self._icontainer = self.get_url()
```

Comment:
Major part will be replaced by __CONU__ projet, so do not spent effort there.
Also `self.jmeno` will be removed after using CONU.
@TomasTomecek what do you thing. Does that make sense to repair it, or just let it as it is, and then remove all own code by CONU?